### PR TITLE
Add pane code samples

### PR DESCRIPTION
**Pull request by FabienLelaquais**
+ Minor doc errors here and there

### CODE REVIEW

File: doc/gui/examples/blocks/pane-anchor.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Python environment and run:
+#     python <script>
+# -----------------------------------------------------------------------------------------
+from taipy.gui import Gui
+
+show_pane = True
+
+page = """
+<|{show_pane}|pane|anchor=top|height=50px|
+Here is a some text that is displayed at the top of the page in a pane.
```
Comment (dinhlongviolin1) [inline doc/gui/examples/blocks/pane-anchor.py]: `a some text` doesnt sound right at all

Comment (FabienLelaquais) [reply doc/gui/examples/blocks/pane-anchor.py]: Ha ha right! Thx

**Review by dinhlongviolin1 — DISMISSED** [review]
LGTM

File: taipy/gui/builder/_element.py [hunk 2]
```diff
@@ -260,14 +260,11 @@ class content(_Control):
...
-            None
+    This pseudo-element can be used in the root page of your application. It is replaced at runtime
+    by the content of the page the user navigates to.
 
-        Examples:
-            - To generate `content`, use:
-               ```
-               content()
-               ```
+    The usage of this peuse-element is descripted in
```
Comment (FredLL-Avaiga) [inline taipy/gui/builder/_element.py]: peuse => pseudo ?

File: taipy/gui/hook.py [hunk 3]
```diff
@@ -10,6 +10,7 @@ class Hook:
 
 
 class Hooks(object, metaclass=_Singleton):
+    """NOT DOCUMENTED"""
```
Comment (FredLL-Avaiga) [inline taipy/gui/hook.py]: I think @dinhlongviolin1 plans to change this to `class _Hooks()`

**Review by FredLL-Avaiga — DISMISSED** [review]
typo

File: taipy/gui/hook.py (hunk 3 above)
Comment (FabienLelaquais) [reply taipy/gui/hook.py]: Indeed.

File: taipy/gui/builder/_element.py (hunk 2 above)
Comment (FabienLelaquais) [reply taipy/gui/builder/_element.py]: 👍

**Review by FredLL-Avaiga — DISMISSED** [review]
No example showing the included open button?

**Review by FredLL-Avaiga — APPROVED** [review]

**Pull request merged**